src/Ztm.Zcoin.Rpc/ZcoinRpcClient.cs

```diff
@@ -353,5 +359,73 @@ static ushort ToNative(PropertyType type)
                     throw new ArgumentOutOfRangeException(nameof(type), type, "The value is not valid.");
             }
         }
+
+        class TransactionInfomation
+        {
+            public uint256 TxId;
+            public Money Fee;
+            public BitcoinAddress SendingAddress;
+            public BitcoinAddress ReferenceAddress;
+            public bool IsMine;
+            public int Version;
+            public int TypeInt;
+            public string Type;
+        }
+
+        async Task<TransactionInfomation> DecodeTransactionAsync(Transaction transaction)
+        {
+            var args = new List<object>()
+            {
+                transaction.ToHex(),
+            };
+
+            // Invoke RPC.
+            var resp = await this.client.SendCommandAsync("exodus_decodetransaction", args.ToArray());
```

Comment:
I think this is a bug. If we don't provide block height to `exodus_decodetransaction` it will use current height. That mean it will always use the latest rules to validate transaction. Even with current implementation that we do this right after submitting transaction it is still a bug due to a gab between submitting and invoking `exodus_decodetransaction` still have chance to have a new block (e.g. new rule come into effect to disallow this transaction).

Reply:
Removed, Use `exodus_gettransaction` instead.